x-pack/plugins/fleet/server/services/package_policy.test.ts

```diff
@@ -1170,7 +1171,776 @@ describe('Package policy service', () => {
     });
   });
 
-  describe('overridePackageInputs', () => {
+  describe('preconfigurePackageInputs', () => {
+    describe('when variable is already defined', () => {
+      it('override original variable value', () => {
+        const basePackagePolicy: NewPackagePolicy = {
+          name: 'base-package-policy',
+          description: 'Base Package Policy',
+          namespace: 'default',
+          enabled: true,
+          policy_id: 'xxxx',
+          output_id: 'xxxx',
+          package: {
+            name: 'test-package',
+            title: 'Test Package',
+            version: '0.0.1',
+          },
+          inputs: [
+            {
+              type: 'logs',
+              policy_template: 'template_1',
+              enabled: true,
+              vars: {
+                path: {
+                  type: 'text',
+                  value: ['/var/log/logfile.log'],
+                },
+              },
+              streams: [],
+            },
+          ],
+        };
+
+        const packageInfo: PackageInfo = {
```

Comment:
it takes a lot of code to set up these objects. could we extract default values to a common place and only override the specific parts for a test?

Reply:
If we refactorize the package policy we will have to refactorize the tests and it will probably a better time to create these fixture functions, what do you think?

Reply:
yes, agree